setup.py

```diff
@@ -146,6 +146,7 @@ def get_git_sha() -> str:
         "impala": ["impyla>0.16.2, <0.17"],
         "kusto": ["sqlalchemy-kusto>=1.0.1, <2"],
         "kylin": ["kylinpy>=2.8.1, <2.9"],
+        "maxcompute": ["pyodps>=0.10"],
```

Comment:
This is a good start, but will only unlock some of the functionality between Maxcompute and Superset.

Ideally, you can create a small, separate `db_engine_spec` just for maxcompute. Here's a quick example of one I made for CrateDB: https://github.com/apache/superset/blob/master/superset/db_engine_specs/crate.py#L32

This will ensure for core features like:

- Time grains (selection of day, week, month, etc) -> resulting in the correct SQL generation
- Better experience in "Add Database" screen
- Better support for timestamp & datetime values all around

Reply:
Is the function of time grains implemented?